pkg/runner/runner.go

```diff
@@ -235,7 +253,7 @@ func CreateClient(cfg *Config, promMetrics *api.Metrics) (*pgclient.Client, erro
 		return nil, fmt.Errorf("elector init error: %w", err)
 	}
 
-	if elector == nil {
+	if elector == nil && !cfg.ReadOnly {
```

Comment:
not related to this PR, but do we have plans for **multiple readers**?

Reply:
I don't think we thought about using multiple readers at the same time, the use-case did not occur yet.